### PR TITLE
Adding overwrite for DWP message

### PR DESCRIPTION
**Pull request by zaparka**
We can now choose if we want to use default DWP monitor or just say that DWP is down/up.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
 
   resource :notifications, only: [:edit, :update]
+  resource :dwp_warnings
```
Comment (leoapost) [inline config/routes.rb]: Can we limit this to only edit and update?

Comment (zaparka) [reply config/routes.rb]: yes, I forget about that one :)

**Comment by leoapost** [comment]
How about the `/healthcheck` endpoint? Don't we want to override that as well?

 ([source code](https://github.com/ministryofjustice/fr-staffapp/blob/master/app/services/health_status.rb#L12))

**Comment by zaparka** [comment]
@leoapost I think that the ```healthcheck``` should reflect the normal check. This is a feature that is aimed at users, not Us.

**Pull request merged**